code/index.html

```diff
@@ -4,27 +4,103 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
+    <link rel="stylesheet" href="./style.css">
     <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
     <!-- video or image as a header is cool :) -->
+   
+<header>
+  <div class="top-section-container">
+    <img class="top-section-logo" src="./Assets/Logo/Perfect gym.png" width="60px" height="60px">
+      <nav class="nav-container">
+        <a class="nav-links" href="Home">Home</a>
+        <a class="nav-links" href="About">About</a>
+        <a class="nav-links" href="News">News</a>
+      </nav>
+  </div>
 
-    <!-- Signup form -->
+  <div class="Hero-container">
+    <h1 class="text-hero">-Come and join our Bootcamp this spring!</h1>
+      <picture>
+        <source srcset="./Assets/Images/bootcamp3.jpg" media="(max-width: 767px)"/>
+        <source srcset="./Assets/Images/bootcamp6.jpg" media="(max-width: 992px)"/>
+        <img class="bootcamp-img " src="./Assets/Images/bootcamp5.jpg" alt="Picture of training"/>
+      </picture>
+  </div>
+</header>
 
+    <!-- Signup form -->
+  <div class="content-container">
+    <form class="form-container" action="https://httpbin.org/anything" method="post">
+      <h3 class="form-title-1">Sign up for Bootcamp</h3>
+      <label for="fname">First name</label><br>
+      <input class="input-form" type="text" name="firstname" required><br>
```

Comment:
We can assign label to input thanks to `for`-`id` attribute connection. Without, it won't work. This is proper example :

```
<label for="fname">First name</label><br>
<input id="fname" class="input-form" type="text" name="firstname" required>
```

Moreover, we shouldn't use `<br />` tags for breaking the line, wrapping piece of code inside `<div />` will give us same result. Let's fix that